internal/action/sync.go

```diff
@@ -69,13 +73,22 @@ func (s *Action) autoSync(ctx context.Context) error {
 
 	ls := s.rem.LastSeen("autosync")
 	debug.Log("autosync - last seen: %s", ls)
-	syncInterval := autosyncIntervalDays
+	syncInterval := autosyncIntervalMinutes
 
-	if s.cfg.IsSet("autosync.interval") {
-		syncInterval = s.cfg.GetInt("autosync.interval")
+	if intervalStr := s.cfg.Get("autosync.interval"); intervalStr != "" {
+		factor := dayInMinutes
```

Comment:
This looks a bit too complicated. I would suggest we just use [`time.ParseDuration`](https://pkg.go.dev/time#ParseDuration) unless it's a plain number. In that case we must handle it as if it is a day to keep backwards compatibility.

Maybe like this:

```
if !strings.ContainsAny(intervalStr, "hmsd") {
  intervalStr += "d"
}
syncInterval = time.ParseDuration(intervalStr)
```

Reply:
Quite nice, I did not know about `time.ParseDuration`. But I see that it does not support `d` for day. So if we should do it this way, I guess we need a little helper that adds support for `d`, and suffix-less for backwards compat. What do you say?

Reply:
Ahw, you are right. We could use something like this maybe? https://github.com/xhit/go-str2duration

Reply:
We could, I've pushed another commit attempting that method. I guess it is more featureful, but adding another dependency counts for quite a lot of added complexity to me. But maybe it's fine?